### DISCUSSION TITLE
월드 개발할 때 디센트럴랜드처럼 Opensea 사이트로 연동이 되거나 외부 웹사이트로 연동/연결이 되도록 하는 기능이 있을까요?

### DISCUSSION THREAD

**Discussion by kimchichopchop** [naverz/zepeto-studio-kor#1251] [5. Q&A]
월드 개발할 때 디센트럴랜드처럼 Opensea 사이트로 연동이 되거나 외부 웹사이트로 연동/연결이 되도록 하는 기능이 있을까요?

**Comment by zlmrzlo** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/865
지금 현재 상황에서 유니티로 개발하실 경우 제페토와 제휴를 맺지 않을 경우 외부 연결 기능을 이용할 수 없는 것 같네요.

**Reply by kimchichopchop** [reply]
답변 감사합니다
